LICENSE.md

```diff
@@ -1,20 +1,57 @@
-### WordPress - Web publishing software
+## Gutenberg
 
     Copyright 2011-2021 by the contributors
 
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation; either version 2 of the License, or
-(at your option) any later version.
+**License for Contributions (on and after April 15, 2021)**
 
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
+All code contributed to the Gutenberg project is dual-licensed, 
+and released under both of the following licenses: 
 
-You should have received a copy of the GNU General Public License
-along with this program; if not, write to the Free Software
-Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+the GNU General Public License as published by the Free Software Foundation; 
+either version 2 of the License or (at your option) any later version (the “GPL”)
+and the Mozilla Public License, Version 2.0 (the “MPL”). 
+
+**Project License (as of April 15, 2021)**
```

Comment:
> License for Contributions

and

> Project License

I wonder, are those 2 a different thing to each other? Or is it just a matter of establishing the timeline? If only for the timeline, can we use the same title for both sections? 🤔

Reply:
Oh, I think I understand the distinction after re-reading it: one is about only the new contributions while the other is about the whole project.

I guess I have a new question:

> **Project License (as of April 15, 2021)**

Do we need that date mention there? I mean, the project license will continue as-is anyway until permission to dual-license past contributions is also granted so, not sure if the April 15th is denoting something 🤔

Reply:
> Do we need that date mention there? I mean, the project license will continue as-is anyway until permission to dual-license past contributions is also granted so, not sure if the April 15th is denoting something

I mostly agree with you here. My only hesitation about removing that date is that I think it then becomes bit ambiguous whether the "License for new contributions on or after April 15" supersedes or otherwise alters the project license. With that said, I would like to remove that date because having it there does imply that the project license is changing on April 15 when it is actually staying the same. Not sure the best way to do that yet though. 🤔

Reply:
Went ahead and updated the license with a change along ^those^ lines.

Reply:
Changes work for me! 👍